app/src/androidTest/java/org/mozilla/fenix/ui/LibraryMenuTest.kt

```diff
@@ -0,0 +1,63 @@
+package org.mozilla.fenix.ui
+
+import org.junit.Rule
+import org.junit.Test
+import org.mozilla.fenix.helpers.HomeActivityTestRule
+import org.mozilla.fenix.ui.robots.homeScreen
+
+class LibraryMenuTest {
+
+    @get:Rule
+    val activityTestRule = HomeActivityTestRule()
+
+    @Test
+    fun libraryMenuItemsTest() {
+        homeScreen {
+        }.openThreeDotMenu { }
```

Comment:
the desired syntax for transitions is slightly different....
```
}.openThreeDotMenu {
}.openLibrary {
```
The brackets around transition functions are always on the same line and always face outward for readability.
(I know, it looks really weird 🤷‍♂ )